### ISSUE TITLE
GET /groups/{id}/members returns 403

### ISSUE THREAD

**Issue by Torvin** [microsoftgraph/microsoft-graph-docs#3829]
I'm trying to get a list of users of a teams group. The [documentation says](https://docs.microsoft.com/en-us/graph/api/group-list-members?view=graph-rest-1.0) that I need to have one of these delegated permissions in order to get it: 

`User.ReadBasic.All, User.Read.All, Directory.Read.All, Directory.AccessAsUser.All`. 

My app has `User.Read` and `User.ReadBasic.All` permissions and when I acquire a token, I ask for this scope: `https://graph.microsoft.com/User.ReadBasic.All` and when I get the token back I can see it:

`"scp": "openid profile User.Read User.ReadBasic.All email",`

And yet, when I'm trying to call that API with the acquired token, I get this response:

```
HTTP/1.1 403 Forbidden
Cache-Control: private
Transfer-Encoding: chunked
Content-Type: application/json
request-id: 40d5384e-aadb-460c-9583-19f040795e0c
client-request-id: 40d5384e-aadb-460c-9583-19f040795e0c
x-ms-ags-diagnostic: {"ServerInfo":{"DataCenter":"Southeast Asia","Slice":"SliceC","Ring":"2","ScaleUnit":"000","Host":"AGSFE_IN_2","ADSiteName":"SEA"}}
Access-Control-Allow-Origin: *
Access-Control-Expose-Headers: ETag, Location, Preference-Applied, Content-Range, request-id, client-request-id
Duration: 36.278
Strict-Transport-Security: max-age=31536000
Date: Sun, 20 Jan 2019 23:22:30 GMT

{
  "error": {
    "code": "Authorization_RequestDenied",
    "message": "Insufficient privileges to complete the operation.",
    "innerError": {
      "request-id": "40d5384e-aadb-460c-9583-19f040795e0c",
      "date": "2019-01-20T23:22:30"
    }
  }
}
```

Could somebody please clarify if I'm doing something wrong or is the documentation wrong about the permissions?

**Comment by Torvin** [comment]
PS: when I request a `Directory.AccessAsUser.All` permission - it works perfectly fine, so I'm pretty sure I'm sending the request correctly

**Comment by FaithOmbongi** [comment]
Thank you @Torvin for highlighting the permission issue. The issue was also raised up in #10302, since resolved via PR #10459.
No additional updates identified for the page. Closing the issue.
